news/6386.bugfix

```diff
@@ -0,0 +1 @@
+Support running Subversion client interactively on Subversion 1.8+.
```

Comment:
I think you can be a little more complete in the description here and say something like, "Extend to Subversion 1.8+ the behavior of calling Subversion in interactive mode when pip is run interactively." This way it's clearer that (1) this behavior is present < 1.8, and (2) the user doesn't have to do anything to get this behavior.